orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -5334,3 +5334,12 @@ def run_step(model, x):
     _test_helpers.assert_values_are_close(pt_prediction, ort_prediction)
     _test_helpers.assert_values_are_close(pt_loss, ort_loss)
     _test_helpers.assert_values_are_close(pt_x.grad, ort_x.grad)
+
+
+def test_eval_model_mod():
```

Comment:
typo here: test_eval_model_mod , should be test_eval_model_mode

Reply:
thx, done